armsrc/lfops.c

```diff
@@ -1539,7 +1797,7 @@ void ReadPCF7931() {
         tries++;
         if (BUTTON_PRESS()) return;
     } while (num_blocks != max_blocks);
-end:
+ end:
```

Comment:
Actually it is needed in function ReadPCF7931.
I just added a space in front of it so I could hide the code properly in my editor...